src/app/services/services-api.service.ts

```diff
@@ -68,4 +77,14 @@ export class ServicesApiService {
       map(response => response),
       catchError(error => throwError(error)));
   }
+
+  /**
+   * POST | /foglamp/plugin?action=install
```

Comment:
you mean service?

Reply:
Yes it is for service but endpoint is `/foglamp/plugin?action=install`